CHANGES.md

```diff
@@ -21,7 +21,10 @@
   - Add new package for `irmin` command-line tool (#1951, @metanivek)
 
 - **irmin-pack**
-  - Add a GC (TODO)
+  - Add a garbage collection feature, allowing a user to discard commits older
+    than a specified commit.  This feature is only enabled for stores that use
+    `Indexing_strategy.minimal`. See `examples/gc.ml` for a demonstration of
+    usage.
```

Comment:
Could you also add the name of the new function? so that a quick search on the README will help to know in which version this has been introduced. Thanks

Reply:
Ah, good idea. The function names are sort of generic so perhaps a modification like:

"The primary API is `Store.Gc.run`, `Store.Gc.wait`, and `Store.Gc.is_finished`. See `examples/gc.ml` for a demonstration of usage."

Does this look good for what you have in mind?

Reply:
yes perfect!